api.go

```diff
@@ -185,6 +187,102 @@ func (api *API) Spec() (spec *openapi3.T, err error) {
 	return
 }
 
+// the interface used by function SpecWithOauth2
```

Comment:
```suggestion
// the interface used by function [*API.SpecWithOauth2]
```

You could use navigation in godoc with this.